src/getdependencies.ts

```diff
@@ -1,24 +1,44 @@
 import { error } from 'console';
+const readYaml =  require('read-yaml');
 
-let readJson;
+let readJson, parseYaml;
 let dependencies = [];
 
-export function getDependencies(repo: string, packageScope: string[]) {
+export function getDependencies(repo: string, packageScope: string[], repoMap: Map<string,string>) {
   try {
-    let path = '../repos/' + repo + '/package.json';
-
+    let path = '../' + repoMap.get(repo) + '/' + repo + '/package.json';
     dependencies = [];
     readJson = require(path).dependencies;
     for (const j in readJson) {
-      for (const scope of packageScope) {
-        if (j.includes(scope)) {
-          dependencies.push(j + ': ' + readJson[j]);
+      if (packageScope.length === 0) {
+        dependencies.push(j + ': ' + readJson[j]);
+      }
+      else {
+        for (const scope of packageScope) {
+          if (j.includes(scope)) {
+            dependencies.push(j + ': ' + readJson[j]);
```

Comment:
So this is setting a version after the `:` right? Usually this is denoted as `${packageName}@${versionSemVar}` E.G. `@lifeomic/attempt@^1.0.0`

Reply:
I see. I was trying to keep it consistent with how the package.json shows it